e2e_tests/tests/experiment/experiment.py

```diff
@@ -13,11 +13,16 @@
 import requests
 
 import determined_common.api.authentication as auth
+from determined import experimental
 from determined_common import api, yaml
 from tests import cluster
 from tests import config as conf
 
 
+def native_experiment_submit(context_dir, configuration, follow_logs=False):
+    experimental.submit(context_dir=context_dir, config=configuration, follow_logs=follow_logs)
```

Comment:
blocking: need to specify master_url here which you can get via `conf.make_master_url()`.

Reply:
addressed in recent commit